cmd/juju/application/deployer/series_selector_test.go

```diff
@@ -166,16 +166,6 @@ func (s *SeriesSelectorSuite) TestCharmSeries(c *gc.C) {
 			},
 			err: "series: disco not supported",
 		},
-		{
-			title: "juju deploy multiseries with force  # use model series defaults if supported by charm, force",
```

Comment:
can we adapt this test instead of deleting it? are we losing coverage?

Reply:
The test scenario is juju deploy with --force --series `an-unsupported-old-series`.
One of the changes of this PR is to disallow/reject this scenario.
So this test does not make sense anymore.